packages/common/src/utils/signature_utils.ts

```diff
@@ -23,10 +23,10 @@ export class SignatureUtils {
     commitData: CommitData,
     did: DID,
     controller: string,
-    family: string,
+    model: StreamID,
```

Comment:
```suggestion
    model: StreamID | null,
```

Reply:
since TileDoc passes in null

Reply:
also, there is a `@param` JSDoc above, that has to be changed